docs/userguide/tasks.rst

```diff
@@ -1440,6 +1440,16 @@ The default value is the class provided by Celery: ``'celery.app.task:Task'``.
 Handlers
 --------
 
+.. method:: before_start(self, task_id, args, kwargs)
+
+    Run by the worker before the task starts executing.
```

Comment:
you can add added in version 5.2 notation

Reply:
Added the directive in the current file.
I have noticed that the notation is sometimes added in the docstring of the source code. Should I add it there as well or is it not needed?

Reply:
It'd be nice to have both, yes.

Reply:
Added in docstring as well.